### DISCUSSION TITLE
How do I define a keyboard shortcut to change the 'mpv options' field?

### DISCUSSION THREAD

**Discussion by hheimbuerger** [smplayer-dev/smplayer#737] [Q&A]
I constantly need to change the `options` field for MPlayer/mpv:
![image](https://github.com/smplayer-dev/smplayer/assets/159414/d3ce13a6-d65c-43b7-b6e8-f95d9d6b0aaf)

Is there a way to define a keyboard shortcut to set this field to a specific string (and then trigger a reload if necessary, as clicking on 'OK' in that dialog would)?

**Comment by smplayer-dev** [comment]
I'm afraid there isn't.

**Reply by hheimbuerger** [reply]
I saw [Lua scripting](https://github.com/mpv-player/mpv/blob/master/DOCS/man/lua.rst) to be mentioned in the mpv docs. Is this rabbit hole worth going down for me, or is that just a no-go or irrelevant path when using mpv through SMPlayer?

Any other workarounds/hacks that I could look into? Or is it down to modifying the SMPlayer source?
